### DISCUSSION TITLE
Custom Settings table name

### DISCUSSION THREAD

**Discussion by ancient-spirit** [spatie/laravel-settings#71] [Ideas]
Can the Settings model table name made custom by defining it on config/settings.php ?

I have use case for it and I will appreciate if the custom Setting model name can be changed

**Comment by rubenvanassche** [comment]
Hi @oxyrealm-dev this was added in v2.1.6: https://github.com/spatie/laravel-settings#database-repository

**Reply by ancient-spirit** [reply]
thank you very much. much appreciated
